docs_user/modules/proc_migrating-mon-and-mgr-from-controller-nodes.adoc

```diff
@@ -394,17 +363,9 @@ And at this point the cluster is clean:
     usage:   43 MiB used, 400 GiB / 400 GiB avail
     pgs:     1 active+clean
 ----
-
-oc0-controller-1 has been removed and powered off without leaving traces on the ceph cluster.
-
++
+The `oc0-controller-1` is removed and powered off without leaving traces on the Red Hat Ceph Storage cluster.
++
 The same approach and the same steps can be applied to migrate oc0-controller-2 to oc0-ceph-1.
+//kgilliga: Are "oc0-controller-2 to oc0-ceph-1" generic names? Can we say something like: "Repeat this procedure for additional controller and Red Hat Ceph Storage nodes in your environment?"
```

Comment:
Yes, I would say that the same procedure should be repeated for the other controller nodes until all the mon/mgrs are migrated to the target nodes.